Spain flavour

ArnauGit opened the pull request:
#Failing to reform should trigger a revolution (should be trying to enact Council Rep.) /and trying to avoid new tag.

Arcadyi committed:
Merge pull request #593 from Cold-War-Project/dev

Patch 0.2.9

ArnauGit committed:
Update Spanish laws (now within law_package)

Nationalize Public Utility Companies
Religious Nationalism
Administrative
Supremacist
#Ruling IGs need to be revised, sometimes get triggered, sometimes not

ArnauGit committed:
Spain - Edit IGs

Changed code for government IGs
Added ideas

ArnauGit committed:
Added Carrero Blanco for Spain

Carrero Blanco as securocrat leader
Franco gains Caudillo trait

ArnauGit committed:
Caudillo trait

Added Caudillo trait for Franco

ArnauGit committed:
Add localization for Caudillo trait

Matti-96 committed:
feat: added francoism modifier to adjust ig clout

Added a timed modifier called Francoism, which helps set up Spain with a contested government that is somewhat legitimate, by modifying the clout of the 3 interest groups in power and suppressing the other interest groups. The modifier currently lasts for 10 years.

Matti-96 committed:
fix: character name fix

fixed the issue with character names not appearing when using the create_character effect

ArnauGit committed:
Spanish IG leaders added

Julio Muñoz Ramonet - Entrepreneur
Raimundo Llopis - Laborists
Enrique Pla - Moralists
Jose Ortega y Gasset - Intelligentsia
Raimundo Fernandez-Cuesta - Patriots

ArnauGit committed:
Added Spanish IG names

ArnauGit committed:
Update names_i_ for Spanish IG

ArnauGit committed:
Edited Francoism nation modifiers

Stronger patriots

ArnauGit committed:
New path for Caudillo gfx

ArnauGit committed:
Add files via upload

New path for CWP made icon characters

ezappala committed:
Merge branch 'dev' into Update-of-Spanish-laws

Commit:
Merge branch 'dev' into Update-of-Spanish-laws

ArnauGit committed:
JE entry and pariah for Spain

ArnauGit committed:
JE for Francoist reform

ArnauGit committed:
International pariah loc

ArnauGit committed:
Francoist reform loc

ArnauGit committed:
International pariah modifier

ArnauGit committed:
Decreased relations for Spain

ArnauGit committed:
UN application not possible with International Pariah modifier

ArnauGit committed:
Spain events file

ArnauGit committed:
Delete events/spain_events directory

ArnauGit committed:
Spain events file corrected

ArnauGit committed:
00_code_on_actions for Spanish event

ArnauGit committed:
Spanish state buildings balanced

ArnauGit committed:
Spain event image

ArnauGit committed:
Image loc

ArnauGit committed:
Spanish reform fail rebalance

ArnauGit committed:
Update CWP_francoist_reform.txt

ArnauGit committed:
Added Franco-Peron protocols decision and event

ArnauGit committed:
Added Peronism modifier for Argentina

ArnauGit committed:
Added Juan Perón as Argentina's laborist leader

ArnauGit committed:
Added image for Franco Peron Protocols

ezappala committed:
Merge branch 'dev' into Update-of-Spanish-laws

DZappala commented on localization/english/CWP/argentina/peronism_l_english.yml:
```diff
@@ -0,0 +1,6 @@
+﻿l_english:
+franco_peron_event.1.t: "Franco-Perón Protocols"
+franco_peron_event.1.d: "The possibility of Argentina's Peronist government assisting the isolated Spain by sending grain and other agricultural products involves a multifaceted consideration of diplomatic, economic, and political factors. This assistance could strengthen diplomatic ties, generate revenue for Argentina's economy, and stimulate its agricultural sector. However, it would probably a ...
+franco_peron_event.1.f: "Maybe Franco is the demonstration of a political third-way."
+franco_peron_event.1.a: "Let's send the help."
+franco_peron_event.2.a: "We should take care of ourselves."
```
```suggestion
﻿l_english:
franco_peron_event.1.t: "The Franco-Perón Protocol"
franco_peron_event.1.d: "Following years of wartime, Spain is in need of our help. Perhaps it's time we extend an olive branch to our homeland and their powerful leader."
franco_peron_event.1.f: "Involving a multifaceted scheme of diplomatic, economic and political considerations, Argentina's Perónist government decides whether or not to assist an isolated Spain by way of agricultural aid. This assistance could strengthen diplomatic ties, generate revenue for Argentina's economy, and stimulate its agricultural sector. However, given Franco's political perception, drifting from their sphere will anger the US. A complex decision faces Perón's government."
franco_peron_event.1.a: "Let's send them aid."
franco_peron_event.2.a: "We should take care of ourselves."
```

DZappala replied:
some various edits here to grammar and phrasing. As well, shortened the large text description to include the most relevant information.

DZappala commented on localization/english/CWP/spain/francoist_reform_je_l_english.yml:
```diff
@@ -0,0 +1,11 @@
+﻿l_english:
+francoist_reform: "Francoist Reform"
+francoist_reform_reason: "After the defeat of fascism in the war we need to change some fundamental things to get international recognition. It is time to slowly enforce some minimal reforms, we are not fascists, aren't we...?"
+francoist_reform_goal: "Our ideology was just based on Catholicism afterall. We will use Movimiento Nacional instead of the Party"
```
```suggestion
﻿l_english:
francoist_reform: "Francoist Reform"
francoist_reform_reason: "With the defeat of fascism reforms must be made to garner international recognition. Because we're not fascists... #italic right?#!"
francoist_reform_goal: "Our ideology was just based on Catholicism after all. It's time for a #italic Movimiento Nacional!#!"
```

DZappala commented on localization/english/CWP/spain/francoist_reform_je_l_english.yml:
```diff
@@ -0,0 +1,11 @@
+francoist_reform_reason: "After the defeat of fascism in the war we need to change some fundamental things to get international recognition. It is time to slowly enforce some minimal reforms, we are not fascists, aren't we...?"
+francoist_reform_goal: "Our ideology was just based on Catholicism afterall. We will use Movimiento Nacional instead of the Party"
+
+democracia_organica.t: "Demoracia Orgánica"
+democracia_organica.d: "The organic conception of society dates back to the Stoics, while its foremost theorist was Althusius. In the Contemporary Age, German idealism, with figures like Hegel and lesser-known thinkers like Karl Christian Friedrich Krause, embraced this idea. Krause's disciple, Heinrich Ahrens, systematically developed it and became one of the most dedicated proponents of Krausis ...
+democracia_organica.a: "One Nation One Party"
+democracia_organica.b: "We must turn to FET y de las JONS (Falange Española Tradicionalista y de las Juntas de Ofensiva Nacional Sindicalista) as our state-party."
+democracia_organica.c: "We are a democracy in our way."
+democracia_organica.e: "We will use the Democracia Orgánica to legitimize ourselves as an alternative to liberal democracy. A step for guaranteeing that we distance ourselves from fascism."
```
```suggestion
democracia_organica.t: "Demoracia Orgánica"
democracia_organica.d: "Our nation finds itself at a political crossroads. In which direction should we progress ourselves on the international stage?"
democracia_organica.f: "The organic conception of society dates back to the stoics. While its foremost theorist was Althusius, in the Contemporary Age figureheads such as Hegel and lesser-known thinkers like Karl Christian Friedrich Krause, embraced this political thought. Krause's disciple, Heinrich Ahrens systematically developed it and became one of the most dedicated proponents of Krausist philosophy. In contrast to liberal democracies, organic democracy considered that instead of oft criticized political parties, the natural organs of association were three: the family, where one is born; the municipality, where one lives; and the trade union, where one works."
democracia_organica.a: "One nation, one party."
democracia_organica.b: "Turn to #italic La Falange#! as our guide."
democracia_organica.c: "We are a democracy!"
democracia_organica.e: "Use #italic Democracia Orgánica#! to distance ourselves from fascism."
```

- FET y de las JONS was colloquially reffered to as "La Falange" in spain and is the name of the registered party in spain today. Pointedly I also removed "as our state-party" since franco VEHEMENTLY insisted FET y de las JONS was a "movement", not a party. 

- Moved flowery description to the flavor section and changed the description to be more direct with what we want the player to decide.

ArnauGit commented on common/history/diplomacy/00_relations.txt:
> If you haven't set these already, should Spain also start with a truce with various nations around the globe?

I don't think so, not participated in the war, plus at the moment the idea of allied intervention was still on the air.

ArnauGit committed:
Edited Spanish flavour

Followed Dom's corrections.
Changed the francoist_reform_goal to "Our ideology was just based on Catholicism after all. It's time to turn back to Falangism!" as more suitable.

ArnauGit committed:
Merge branch 'Update-of-Spanish-laws' of https://github.com/Cold-War-Project/CWP-Main into Update-of-Spanish-laws

ArnauGit committed:
Edited ideologies
